demo/summer-2016.html

```diff
@@ -240,23 +240,13 @@
     });
 
 
+    require(["ace/autocomplete/papeeria/tex_completer"], function(completer) {
```

Comment:
I think about lib/ace/mode/papeeria/tex_completions.coffee because  I will modify papeeria_latex.js. What is you opinion?